modules/quality_control/php/quality_control.class.inc

```diff
@@ -119,8 +119,15 @@ class Quality_Control extends \NDB_Menu_Filter
                   LEFT JOIN mri_protocol mp ON mp.Scan_type=".$acqID."
                   WHERE
                       ts.TarchiveID = t.TarchiveID
-                      AND (ts.EchoTime BETWEEN mp.TE_min AND mp.TE_max)
-                      AND (ts.RepetitionTime BETWEEN mp.TR_min AND mp.TR_max)".
+                      AND 
+                          (ts.EchoTime BETWEEN 
+                                SUBSTRING_INDEX(mp.TE_range,'.',1) AND
+                                SUBSTRING_INDEX(mp.TE_range,'.',2)
```

Comment:
@kongtiaowang I don't understand why there are SUBSTRING_INDEX here? Could you explain it to me? Thanks!

Reply:
@kongtiaowang TE_range and TR_range do not exist anymore in the mri_protocol table. It should be TE_min, TE_max and TR_min TR_max as per this PR: https://github.com/aces/Loris/pull/3973

Reply:
@cmadjar @driusan That is very helpful. We don't use TE_rang anymore.